pkg/kubelet/kubelet.go

```diff
@@ -1900,7 +1919,11 @@ func (kl *Kubelet) syncPod(o syncPodOptions) error {
 	}
 
 	// Wait for volumes to attach/mount
-	if err := kl.volumeManager.WaitForAttachAndMount(pod); err != nil {
+	defaultedPod, _, err := kl.defaultPodLimitsForDownwardApi(pod, nil)
```

Comment:
commenting for future reviewer clarity, I removed 'lastUpdatedNode` altogether from the orginal version of this PR to keep change set minimal at this point in the release.